src/content/app/tools/blast/state/blast-results/blastResultsSlice.ts

```diff
@@ -60,7 +61,11 @@ export type BlastResult = {
   sequenceId: number;
   genomeId: string;
   status: JobStatus;
-  data: null; // TODO: add data type
+  data: null | BlastJobResultsFromAPI;
+};
+
+export type BlastResultWithData = Omit<BlastResult, 'data'> & {
+  data: NonNullable<BlastResult['data']>;
```

Comment:
Is there any difference between `data: BlastJobResultsFromAPI` & `data: NonNullable<BlastResult['data']>`?

Reply:
None.

Reply:
Updated the name. Both here and more generally renamed the `BlastResult` type to `BlastJob`